src/Utils/CoverVendor/VendorImageItem.php

```diff
@@ -7,11 +7,13 @@
 
 namespace App\Utils\CoverVendor;
 
+use App\Entity\Vendor;
+
 class VendorImageItem implements \Stringable
 {
     private bool $found = false;
     private bool $updated = false;
-    private string $vendor;
+    private Vendor $vendor;
```

Comment:
Is vendor image item ever sent into the queue system and will this impact the required memory to handle the message?

Reply:
No - the only non-primitive type used in any of the messages is `Material` in `IndexItem`